test/AWSUtils.Tests/Streaming/SQSStreamTests.cs

```diff
@@ -48,14 +48,15 @@ protected override void ConfigureTestCluster(TestClusterBuilder builder)
                     {"DataConnectionString", $"Service={AWSTestConstants.Service}"},
                     {"DeleteStateOnClear", "true"}
                 };
-                options.ClusterConfiguration.Globals.RegisterStorageProvider<DynamoDBStorageProvider>("DynamoDBStore", storageConnectionString);
-                var storageConnectionString2 = new Dictionary<string, string>
-                {
-                    {"DataConnectionString", $"Service={AWSTestConstants.Service}"},
-                    {"DeleteStateOnClear", "true"},
-                    {"UseJsonFormat", "true"}
-                };
-                options.ClusterConfiguration.Globals.RegisterStorageProvider<DynamoDBStorageProvider>("PubSubStore", storageConnectionString2);
+                // FIXME: How to configure TestClusterBuilder with the extension methods
```

Comment:
Why is this commented out? What's here to fix?

Reply:
I didn't found how to properly use the extension methods to register the provider using the `TestCluster`.

Reply:
Updated this to setup the providers correctly (I think).

@galvesribeiro, can vet that is setup the secrets correctly?  There isn't a data connection string in the new options..?